src/views/fragments/IntroductionContent/index.jsx

```diff
@@ -0,0 +1,61 @@
+import { useNavigate,useLocation } from "react-router-dom";
```

Comment:
please remember to use prettier formatter when working on the code, this is mentioned in the HowTo doc